src/components/Input/interface.ts

```diff
@@ -0,0 +1,13 @@
+export interface SearchInputProps {
+  suggestions: Array<string>;
+}
+
+export interface SuggestionsProps {
+  filteredSuggestions: Array<string>;
+  activeSuggestionIndex: number;
+  chooseSuggestion: (e: React.MouseEvent<HTMLLIElement, MouseEvent>) => void;
```

Comment:
But those are still suggestions, I think - we can talk this through on the call.

Reply:
Yes, let's talk about it :)